### DISCUSSION TITLE
Conflicts with installation of thorium web browser

### DISCUSSION THREAD

**Discussion by viasux** [edrlab/thorium-reader#2638] [Developer corner]
I believe this installs to the same location that the web browser does, so it would be nice if the installation name could be thorium-reader instead of thorium, or if a zip release could be added.

**Comment by danielweck** [comment]
Ah, that is indeed annoying! 

https://github.com/edrlab/thorium-reader/blob/9d2e3e840425b9d35292f036167148c0f1812f9f/package.json#L247

https://thorium.rocks

...my concern is that renaming the Thorium Reader executable on Linux (i.e. `thorium` which is currently distributed via a Debian package / installer, though it is worth noting that the AppImage has its own distinct executable name) would probably break things on the consumer side, i.e. in any deployment script that depends on the current filename.

For example:
https://gitlab.com/mtyshibata/thorium-reader-snap/-/blob/main/snap/snapcraft.yaml?ref_type=heads#L19

**Reply by viasux** [reply]
> Ah, that is indeed annoying!
> 
> https://github.com/edrlab/thorium-reader/blob/9d2e3e840425b9d35292f036167148c0f1812f9f/package.json#L247
> 
> https://thorium.rocks
> 
> ...my concern is that renaming the Thorium Reader executable on Linux (i.e. `thorium` which is currently distributed via a Debian package / installer, though it is worth noting that the AppImage has its own distinct executable name) would probably break things on the consumer side, i.e. in any deployment script that depends on the current filename.
> 
> For example: https://gitlab.com/mtyshibata/thorium-reader-snap/-/blob/main/snap/snapcraft.yaml?ref_type=heads#L19

Ah that makes sense, maybe just change it for windows, and add a zip release for now?

**Comment by danielweck** [comment]
Related: https://github.com/edrlab/thorium-reader/discussions/2682

**Comment by viasux** [comment]
Hey there! I was curious if there's going to be a supported alternate installation method (ex: a zip file) or a proposed fix for this. Is this a wontfix? Or is it just something that will be dealt with eventually when there's time?

Thank you for your hard work!

**Reply by danielweck** [reply]
Given our current priorities (we are a tiny dev team) I fear that this issue must be pushed back a bit. We have an important release coming up (first since the initial major Thorium 3.0) and I am concerned that the change of executable name will cause regressions on the consumer side as well as in our build pipeline.
